README.md

```diff
@@ -1,344 +1,49 @@
-    end
-end
-```
-## CICD
-### Jenkins setup
-<img src="https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/jenkins_tfswitch.png" alt="drawing" style="width: 170px;"/>
-
-```sh
-#!/bin/bash 
-
-echo "Installing tfswitch locally"
-wget https://raw.githubusercontent.com/warrensbox/terraform-switcher/release/install.sh #Get the installer on to your machine
-
-chmod 755 install.sh            #Make installer executable
-
-./install.sh -b `pwd`/.bin      #Install tfswitch in a location you have permission
-
-CUSTOMBIN=`pwd`/.bin            #set custom bin path
-
-export PATH=$PATH:$CUSTOMBIN    #Add custom bin path to PATH environment
-
-$CUSTOMBIN/tfswitch -b $CUSTOMBIN/terraform 0.11.7 #or simply tfswitch -b $CUSTOMBIN/terraform 0.11.7
-
-#OR 
-
-$CUSTOMBIN/tfswitch -d 0.11.7 -b $CUSTOMBIN/terraform  #or simply tfswitch -d 0.11.7 -b $CUSTOMBIN/terraform 
-
-terraform -v                    #testing version
-```
-
-### Circle CI setup
-
-<img src="https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/circleci_tfswitch.png" alt="drawing" style="width: 280px;"/>
-
-
-Example config yaml
-```yaml
-version: 2
-jobs:
-  build:
-    docker:
-      - image: ubuntu
-
-    working_directory: /go/src/github.com/warrensbox/terraform-switcher
-
-    steps:
-      - checkout
-      - run: 
-          command: |    
-            set +e   
-            apt-get update 
-            apt-get install -y wget 
-            rm -rf /var/lib/apt/lists/*
-
-            echo "Installing tfswitch locally"
-
-            wget https://raw.githubusercontent.com/warrensbox/terraform-switcher/release/install.sh #Get the installer on to your machine
-
-            chmod 755 install.sh            #Make installer executable
-
-            ./install.sh -b `pwd`/.bin      #Install tfswitch in a location you have permission
-
-            CUSTOMBIN=`pwd`/.bin            #set custom bin path
-
-            export PATH=$PATH:$CUSTOMBIN    #Add custom bin path to PATH environment
-
-            $CUSTOMBIN/tfswitch -b $CUSTOMBIN/terraform 0.11.7 #or simply tfswitch -b $CUSTOMBIN/terraform 0.11.7
-
-            #OR
-            $CUSTOMBIN/tfswitch -d 0.11.7 -b $CUSTOMBIN/terraform  #or simply tfswitch -d 0.11.7 -b $CUSTOMBIN/terraform 
-            
-            terraform -v                    #testing version
-```
-## Order of precedence
-
-| Order | Method |
-| --- | ----------- |
-| 1 | .tfswitch.toml |
-| 2 | .tfswitchrc |
-| 3 | .terraform-version |
-| 4 | Environment variable |
-
-With 1 being the highest precedence and 4 the lowest   
-*(If you disagree with this order of precedence, please open an issue)*
-## How to contribute    
+## Installation
+`tfswitch` is available as a binary and on various package managers (eg. brew). 
+
+See [our installation documentation](https://tfswitch.warrensbox.com/Installation) for more details. 
+
+> [!IMPORTANT]
+> The version identifier of `tfswitch` has changed to align with the common `go` package versioning. 
```

Comment:
What are those trailing single spaces for? 🤔

Reply:
There are many in this file - I just split the lines up. I'll remove them.